sdk/lib/collection/linked_hash_map.dart

```diff
@@ -6,19 +6,81 @@ part of dart.collection;
 
 /// A hash-table based implementation of [Map].
 ///
-/// The insertion order of keys is remembered,
+/// __The insertion order of keys is remembered__,
 /// and keys are iterated in the order they were inserted into the map.
 /// Values are iterated in their corresponding key's order.
 /// Changing a key's value, when the key is already in the map,
 /// does not change the iteration order,
 /// but removing the key and adding it again
 /// will make it be last in the iteration order.
```

Comment:
Is it worth having an example here, for this behaviour?

Reply:
updated the example where the item is in initial list and removed then added later back to list and it is last item